src/AppRouter.js

```diff
@@ -123,10 +123,13 @@ const AppRouter = () => (
                     <CompanyOffersManagementPage />
                 </PageLayout>
             </ProtectedRoute>
-            <Route
+            <ProtectedRoute
                 exact
                 path="/offer/:id/edit"
                 key="/offer/:id/edit"
+                unauthorizedRedirectPath="/"
+                unauthorizedRedirectMessage="You are not allowed to edit offers"
+                authorize={(user) => !!(user?.company || user?.isAdmin)}
```

Comment:
Only the owner company can edit the offer, not any user

Reply:
I'm verifying this in [here](https://github.com/NIAEFEUP/nijobs-fe/blob/3e18cf0b864170103cffaf2fd08c4a40d5cfb9e8/src/components/Offers/Edit/EditOfferForm.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L48) and [here](https://github.com/NIAEFEUP/nijobs-fe/blob/3e18cf0b864170103cffaf2fd08c4a40d5cfb9e8/src/components/Offers/Edit/EditOfferForm.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L139-L143), redirecting with the snackbar warning